### PR TITLE
[Asset] Remove VR Preview

### LINKED ISSUE #14111: [Asset] Remove VR Preview
Reasons: 
- vrview library is dead
- image vr preview is not working anyway 😉 , it was accidentally by #3979 

Perfect candidate for a bundle if someone still needs that functionality.

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request  
Resolves #14111

### CODE REVIEW

**Comment by fashxp** [comment]
we need to deprecate that feature in Pimcore 10.6 too

**Comment by brusch** [comment]
There's nothing really to deprecate is it? 
We can add a note in the UI that this feature is going to be removed in 11.x but I guess that's not really a benefit ...

**Comment by lukmzig** [comment]
@brusch we decided to do those small changes, PR is already open :) https://github.com/pimcore/pimcore/pull/14237

File: bundles/AdminBundle/public/js/pimcore/asset/image.js [hunk 1]
```diff
@@ -427,9 +403,6 @@ pimcore.asset.image = Class.create(pimcore.asset.asset, {
 
             this.previewContainerId = 'pimcore_asset_image_preview_' + this.id;
             this.previewMode = 'image';
```
Comment (Corepex) [inline bundles/AdminBundle/public/js/pimcore/asset/image.js]:
```suggestion
```

Should be obsolete ...

Comment (Corepex) [reply bundles/AdminBundle/public/js/pimcore/asset/image.js]: same for https://github.com/pimcore/pimcore/blob/b54d1a6a961f8a7e07de9a01deca1593b8a6a18d/bundles/AdminBundle/public/js/pimcore/asset/image.js#L435

**Review by Corepex — COMMENTED** [review]
This occurrences should also be removed, i guess:

https://github.com/pimcore/pimcore/blob/b54d1a6a961f8a7e07de9a01deca1593b8a6a18d/bundles/AdminBundle/public/js/pimcore/asset/video.js#L116-L120

https://github.com/pimcore/pimcore/blob/b54d1a6a961f8a7e07de9a01deca1593b8a6a18d/bundles/AdminBundle/public/js/pimcore/asset/video.js#L327

File: bundles/AdminBundle/public/js/pimcore/asset/image.js (hunk 1 above)
Comment (lukmzig) [reply bundles/AdminBundle/public/js/pimcore/asset/image.js]: @Corepex thank you, these files are removed now, seems like they were indeed not needed anymore

**Comment by Corepex** [comment]
The patched things look good to me. 


Found some more stuff that should be changed:

We should get rid of the "vr" part here:
https://github.com/pimcore/pimcore/blob/fb9079a2dd7c5d1d5c568556c24009485c3a85b0/README.md?plain=1#L73
https://github.com/pimcore/pimcore/blob/fb9079a2dd7c5d1d5c568556c24009485c3a85b0/doc/Development_Documentation/00_Overview/README.md?plain=1#L30

If this is not needed, we could also remove the icon: 
https://github.com/pimcore/pimcore/blob/fb9079a2dd7c5d1d5c568556c24009485c3a85b0/bundles/AdminBundle/public/css/icons.css#L601-L603

**Comment by lukmzig** [comment]
@Corepex new changes are there. Regarding the icon, I removed the CSS class but I would keep the svg as it can be in use by some of the users.

**Comment by Corepex** [comment]
@lukmzig, that's correct - we shouldn't remove the icon (SVG) since its a part of the icon pack 👍

**Pull request merged**